mantidproject/mantid
# Replaced mantidplot with mantid.simpleapi in plot method of CrystalField

Both for CrystalField and CrystalFieldMultiSite the plot method used plotSpectrum from mantidplot. This was replaced with plotSpectrum from mantid.simpleapi. 

For CrystalFieldMultiSite a check for empty arguments was added to prevent out of range errors.


**To test:**

Use the following test scripts for testing. Both should show an empty plot window and no error message.

For CrystalField:

```
from CrystalField import CrystalField

cf = CrystalField('Ce', 'C2v', B20=0.37737, B22=3.9770, Temperature=44)
cf.plot()
```

For CrystalFieldMultiSite:

```
from CrystalField import CrystalFieldMultiSite

cfms = CrystalFieldMultiSite(Ions=['Ce', 'Pr'], Symmetries=['C2v', 'C2v'], Temperatures=[44.0], FWHMs=[1.1])
cfms.plot()
```

Fixes [#31464](https://github.com/mantidproject/mantid/issues/31464) . 

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
